tests/conftest.py

```diff
@@ -15,6 +21,82 @@ def copy_srcdir_to_tmpdir(srcdir, tmp):
     return tmproot
 
 
+def get_abspath(relpath):
+    if relpath and isinstance(relpath, str):
+        abspath = Path(__file__).parent.joinpath(relpath).resolve()
+        return str(abspath)
+    return relpath
+
+
+def test_js(self) -> Dict[str, Any]:
+    cypress_testpath = get_abspath(self.spec_pattern)
+
+    if not cypress_testpath and not (os.path.isabs(cypress_testpath) and os.path.exists(cypress_testpath)):
+        return {
+            "returncode": 1,
+            "stdout": None,
+            "stderr": f"The spec_pattern you provided cannot be found. (spec_pattern: {self.spec_pattern})",
+        }
+
+    js_test_config = {
+        "specPattern": cypress_testpath,
+        "supportFile": get_abspath("js_test/cypress/support/e2e.js"),
+        "fixturesFolder": False,
+        "baseUrl": "http://localhost:65323",
```

Comment:
Does a hardcoded port work with parallel test execution?

If we have 20 js tests, they all would initiate their own cypress-server and block port 65323.
Can we make this somehow a little bit more safe?
E.g. use a range of 5 ports, and select the next one, if the current one is blocked by an already running cypress server?

Or do I see here a problem, where no problem is?